tools/ts_proto/index.bzl

```diff
@@ -8,26 +8,24 @@ def generate_ts_proto_module(name, protofile, visibility = None):
 
     npm_package_bin(
         name = "generate_js_" + name,
-        tool = "@ts_proto_npm//protobufjs-cli/bin:pbjs",
+        tool = "//tools/ts_proto:bundler",
         data = [protofile],
         testonly = True,
         args = [
-            "-t",
-            "static-module",
-            "-w",
-            "commonjs",
             "$(execpath %s)" % protofile,
+            "js",
         ],
         stdout = "generated_" + js_file,
     )
 
     npm_package_bin(
         name = "generate_ts_" + name,
-        tool = "@ts_proto_npm//protobufjs-cli/bin:pbts",
+        tool = "//tools/ts_proto:bundler",
```

Comment:
What's the context for all of these changes? 

1. ProtobufJS CLI is intentionally kept in a separate install as it's only needed during generation and not for the normal development flow.
2. Doesn't protobuf CLI work correctly anymore? Is the result of an update?

Reply:
By using this bundler script, we can leverage the protobufjs-cli bundler, but modify the import statement used so that this works with ESM.  Currently protobufjs-cli does not have an option for generating ESM.

Reply:
Makes sense. I replied in another comment on this.